extra_geom/detectors.py

```diff
@@ -1810,13 +1815,15 @@ class Epix100Geometry(EpixGeometryBase):
     detector_type_name = 'ePix100'
     pixel_size = 50e-6
     inner_pixel_size = 175e-6
-    asic_gap = inner_pixel_size / pixel_size - 1
-    frag_ss_pixels = 352  # rows
+    asic_gap = 2 * (inner_pixel_size - pixel_size) / pixel_size
+    # ePix100 should have 352 pixel per row, but the data we store has 354
+    frag_ss_pixels = 354  # rows
```

Comment:
Can we confirm that this affects all ePix100 detectors we have/will have, and all data we store? I.e. it's not a quirk of one particular detector, or something that e.g. detector calibration might lop off. I don't want to fix the problem in one case and create a problem somewhere else.

As with my comment on #96, I think the priority is to understand exactly where the discrepancy comes from.

Reply:
I forwarded you an email with implementation details, in short it read 708 rows in the device implementation, so all ePix100 have the same data shape. As far as I can tell, the calibration does not change the data shape.

Reply:
Unfortunately, it can be only a temporary solution. At the end of last week experts from SLAC informed us that two alternative geometries exist (I've forwarded the email to you):

1) One module geometry: one monolithic sensor with 4 ASICs. This geometry does not have gaps, only larger pixels on the inner edges of ASICs.

2) Two modules geometry: two monolithic sensors with 2 ASICs each. Each sensor is in the guard ring.

In the first case, there can be no gap, since the manufacturing is very accurate (nm).

In the second case, it is not a pixel gap. Modules are assembled mechanically and can be located with a shift of columns and with a rotation, and not even in the same plane. It requires multi-module geometry.

Reply:
I see, you already know about two layouts

Reply:
These two rows you added make no sense. They cannot be calibrated, and shouldn't be assembled in the frame. They even don't have positions, since they aren't wired to the sensor. They must be excluded from the geometry.

Reply:
We could also check the array shape passed in to EXtra-geom, and if there are 708 rows, trim away the extra 4 to leave the real pixels, as a convenience feature. This would be similar to what we do for PNCCD to allow passing in the 2 modules as a single image.

Reply:
Yes, we could. But is it worth? I suppose that the assembling method will be able to cut off extra rows from provided data but methods that provide positions for assembling will return positions without extra rows. And what to do with `data_coords_to_positions`?

In principle, it could be a convention, but misleading even more.

Maybe we could add an additional method for data normalization? (it also can add dummy dimension) And add corresponding documentation and examples?

Reply:
> They even don't have positions, since they aren't wired to the sensor.

Thanks, I originally thought they could be the larger edge pixels, I've removed that now.